src/DefaultProgressBar.jsx

```diff
@@ -60,7 +60,7 @@ export default withStyles(({ color, unit }) => ({
   },
 
   DefaultProgressBar_progressBar: {
-    backgroundColor: color.teal,
+    backgroundColor: color.focus,
```

Comment:
I think here this should maybe be `color.progressBar` and point back to the teal? I don't know that there's an inherent relationship between focus state and progress bar color and we should the ability to override individually in the theme.